### PR TITLE
Adapt ktlint links in versioned docs of 1.23

### PR DESCRIPTION
**Pull request by marschwar**
#6103 allows to dynamically link to the correct version of the ktlint documentation. The versioned docs for 1.23.0 still pointed to urls that no longer exist. This PR should fix that.

### CODE REVIEW

File: website/versioned_docs/version-1.23.0/rules/formatting.md [hunk 1]
```diff
@@ -71,20 +72,20 @@ See [ktlint docs](https://pinterest.github.io/ktlint/rules/standard/#chain-wrapp
 
 ### ClassName
 
-See [ktlint docs](https://pinterest.github.io/ktlint/rules/standard/#classobject-naming) for
+See [ktlint docs](hhttps://pinterest.github.io/ktlint/0.49.1/rules/standard/#classobject-naming) for
```
Comment (TWiStErRob) [inline website/versioned_docs/version-1.23.0/rules/formatting.md:75]:
```suggestion
See [ktlint docs](https://pinterest.github.io/ktlint/0.49.1/rules/standard/#classobject-naming) for
```

File: website/versioned_docs/version-1.23.0/rules/formatting.md [hunk 2]
```diff
@@ -161,20 +163,20 @@ from the standard rules, make sure to enable just one. The pro of this rule is t
...
 
-See [ktlint docs](https://pinterest.github.io/ktlint/rules/experimental/#fun-keyword-spacing) for documentation.
+See [ktlint docs](https://pinterest.github.io/ktlint/0.49.1/rules/experimental/#fun-keyword-spacing) for documentation.
 
 **Active by default**: Yes - Since v1.23.0
 
 ### FunctionName
 
-See [ktlint docs](https://pinterest.github.io/ktlint/rules/experimental/#function-naming) for
+See [ktlint docs](https://pinterest.github.io/ktlint/0.49.1/rules/experimental/#function-return-type-spacing) for
```
Comment (TWiStErRob) [inline website/versioned_docs/version-1.23.0/rules/formatting.md:172]: Copy-paste?

File: website/versioned_docs/version-1.23.0/rules/formatting.md [hunk 3]
```diff
@@ -238,7 +240,7 @@ See [ktlint docs](https://pinterest.github.io/ktlint/rules/experimental/#if-else
 
 ### IfElseWrapping
 
-See [ktlint docs](https://pinterest.github.io/ktlint/rules/experimental/#if-else-wrapping) for documentation.
+See [ktlint docs](hhttps://pinterest.github.io/ktlint/0.49.1/rules/experimental/#if-else-wrapping) for documentation.
```
Comment (TWiStErRob) [inline website/versioned_docs/version-1.23.0/rules/formatting.md:243]:
```suggestion
See [ktlint docs](https://pinterest.github.io/ktlint/0.49.1/rules/experimental/#if-else-wrapping) for documentation.
```

File: website/versioned_docs/version-1.23.0/rules/formatting.md (hunk 2 above)
Comment (chao2zhang) [inline website/versioned_docs/version-1.23.0/rules/formatting.md:172]:
```suggestion
See [ktlint docs](https://pinterest.github.io/ktlint/0.49.1/rules/experimental/#function-naming) for
```

**Review by cortinico — APPROVED** [review]
LGTM 👍 but you have a merge conflict

**Review by TWiStErRob — APPROVED** [review]
I think I fixed them all, I didn't have time to review all the links again yet.

**Comment by marschwar** [comment]
Thanks for your support. I just clicked them all.

**Pull request merged**